script/node-disabled-tests.json

```diff
@@ -131,9 +131,6 @@
   "parallel/test-webcrypto-export-import-cfrg",
   "parallel/test-webcrypto-keygen",
   "parallel/test-webcrypto-sign-verify-eddsa",
-  "parallel/test-worker-debug",
```

Comment:
@codebytere why are these changed as part of this backport ? Should they be in a separate PR ?

Reply:
they're not - it happened in trop's logic with the backport. I'll remove it.